templates/pages/home.liquid

```diff
@@ -0,0 +1,286 @@
+{% capture body_content %}{{ current_page.body_content }}{% endcapture %}
```

Comment:
@aaronryematawaran let's remove this override, it shouldn't contained fixed data.